wpilib/wpilib/timer.py

```diff
@@ -34,19 +34,17 @@ def getFPGATimestamp():
     @staticmethod
     def getMatchTime():
         """Return the approximate match time.
-        The FMS does not currently send the official match time to the robots.
-        This returns the time since the enable signal sent from the Driver
-        Station.
-        At the beginning of autonomous, the time is reset to 0.0 seconds.
-        At the beginning of teleop, the time is reset to +15.0 seconds.
-        If the robot is disabled, this returns 0.0 seconds.
-
+        The FMS does not currently send the official match time to the robots,
+        but does send an approximate match time. The value will count down
+        the time remaining in the current period (auto or teleop). 
+        The Practice Match function of the DS approximates the behavior seen
+        on the field.
```

Comment:
This last sentence is after the warning upstream (and in DriverStation which it is intended to match).

Reply:
fixed